medicines/web/components/yellow-card/index.tsx

```diff
@@ -25,6 +25,9 @@ const StyledYellowCard = styled.section`
   header {
     padding: 1rem;
     background-color: ${mhraYellow};
+    svg {
+      height: 31px;
```

Comment:
would it change/break anything for this to be in `rem`?

Reply:
No, but we know the image should be 31 pixels tall. I'd need to figure out how many `rem`s that is. In my not-very-strongly-held opinion, `px` makes more sense in this context, but happy to change if you like.